test/common/alerts_mechanism.go

```diff
@@ -305,15 +350,41 @@ func scaleDeploymentConfig(name string, namespace string, replicas int32, client
 		}
 		getErr := client.Get(goctx.TODO(), k8sclient.ObjectKey{Name: name, Namespace: namespace}, deploymentConfig)
 		if getErr != nil {
-			return fmt.Errorf("Failed to get DeploymentConfig %s in namespace %s with error: %s", name, namespace, getErr)
+			return fmt.Errorf("failed to get DeploymentConfig %s in namespace %s with error: %s", name, namespace, getErr)
 		}
 
 		deploymentConfig.Spec.Replicas = replicas
 		updateErr := client.Update(goctx.TODO(), deploymentConfig)
 		return updateErr
 	})
 	if retryErr != nil {
-		return fmt.Errorf("Update failed: %v", retryErr)
+		return fmt.Errorf("update failed: %v", retryErr)
+	}
+
+	return nil
+}
+
+func checkFuseOperatorReplicasAreReady(ctx *TestingContext, t *testing.T) error {
```

Comment:
It might be a little neater if this function accepted `originalOperatorReplicas` as a parameter. That way it wouldn't have to be a global variable, I think. Given that it's in a simple test, it's probably no big deal.

Reply:
@grdryn good idea, I agree. I've changed `originalOperatorReplicas` to be a local var.